judging/scripts/prescribe/generate_prescriptions.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
Again, copy-pasta-ish of generate_predictions.sh but wanted to be backward compatible. If we're feeling aggressive we can merge the two...some day.

Reply:
No need